pull/github.go

```diff
@@ -730,6 +735,7 @@ func backfillPushedAt(commits []*Commit, headSHA string) {
 // if adding new fields to this struct, modify Locator#toV4() as well
 type v4PullRequest struct {
 	Author    v4Actor
+	Title     string
```

Comment:
While I forgot to mention it in the comment here, you also need to modify `Locator.IsComplete` when adding new `v4PullRequest` fields.